client/my-sites/plugins-wpcom/plugin-panel.jsx

```diff
@@ -6,24 +6,48 @@ import { getSelectedSiteId } from 'state/ui/selectors';
 import { getSiteSlug } from 'state/sites/selectors';
 
 import InfoHeader from './info-header';
+
 import StandardPluginsPanel from './standard-plugins-panel';
 import PremiumPluginsPanel from './premium-plugins-panel';
 import BusinessPluginsPanel from './business-plugins-panel';
 
+import {
+		defaultStandardPlugins,
+		defaultPremiumPlugins,
+		defaultBusinessPlugins
+	} from './default-plugins';
+
+/*
+ * Interpolate the given plugin propagating `siteSlug` value
+ */
+const linkInterpolator = siteSlug => plugin => {
```

Comment:
This is good for now, but I think that if we need more data to pass in here we should move to a key-based approach

```js
const linkInterpolator = replacements => plugin => {
	const { descriptionLink: link } = plugin;

	const descriptionLink = Object.keys( replacements )
		.reduce( ( s, r ) => s.replace( r, replacements[ r ] ), descriptionLink );

	return Object.assign( {}, plugin, { descriptionLink } );
}
```